packages/dso-toolkit/libs/bootstrap/_variables.scss

```diff
@@ -564,19 +562,7 @@ $panel-danger-heading-bg:     $state-danger-bg !default;
 //
 //##
 
-//** Padding around the thumbnail image
-$thumbnail-padding:           4px !default;
-//** Thumbnail background color
-$thumbnail-bg:                $body-bg !default;
-//** Thumbnail border color
-$thumbnail-border:            #ddd !default;
-//** Thumbnail border radius
-$thumbnail-border-radius:     $border-radius-base !default;
-
-//** Custom text color for thumbnail captions
-$thumbnail-caption-color:     $text-color !default;
-//** Padding around the thumbnail caption
-$thumbnail-caption-padding:   9px !default;
+// Moved to @dso-toolkit/sources/src/styling/variables/thumbnail
```

Comment:
Door een variable file `thumbnail.scss` te introduceren wek je de indruk dat er een component `thumbnail` bestaat. De variablen die je hebt verhuisd worden echter alleen op `.img-` classnames gebruikt.

Andere plek graag. Waar worden deze variablen nog meer gebruikt? Als ze maar op één plek worden gebruikt mogen ze best in de file zelf staan..

Reply:
Goed punt, ze worden alleen gebruikt bij de `.img-` varianten, daar heb ik ze nu naar toe verplaatst.